cub/test/catch2_test_device_reduce_env.cu

```diff
@@ -261,7 +261,7 @@ C2H_TEST("Device reduce uses environment", "[reduce][device]", requirements)
             policy_t,
             decltype(d_in),
             output_it_t,
-            offset_t,
+            int,
```

Comment:
What is the point of this change?

Reply:
We use `int` as `offset_t` for `DeterministicDeviceReduceSingleTileKernel` as **num_items** are always small.